axelor-production/src/main/resources/views/AppProduction.xml

```diff
@@ -4,7 +4,8 @@
   xsi:schemaLocation="http://axelor.com/xml/ns/object-views http://axelor.com/xml/ns/object-views/object-views_5.4.xsd">
 
   <form name="app-production-config-form" title="App production"
-    model="com.axelor.apps.base.db.AppProduction" canDelete="false" canNew="false" width="large">
+    model="com.axelor.apps.base.db.AppProduction" canDelete="false" canNew="false" width="large"
+    onSave="action-app-production-method-check-outsourcing-disbled-onsave">
```

Comment:
Please fix typo in action name `action-app-production-method-check-outsourcing-disbled-onsave` --> `action-app-production-method-check-outsourcing-disabled-onsave`